library/Vanilla/PermissionsTranslationTrait.php

```diff
@@ -100,6 +101,85 @@ private function consolidatePermissions(array $permissions) {
         return $result;
     }
 
+    /**
+     * Given a permissions object, return its data array in the explicit-scope format.
+     *
+     * @param \Vanilla\Permissions $permissions
+     * @return array
+     */
+    private function formatPermissions(Permissions $permissions) {
```

Comment:
Did this scope exist somewhere else previously? Or is it just being defined for the first time here?

Reply:
Where is the "explicit-scope format" defined? If the definition is here I'd like to see an example of what it's supposed to be and/or unit tests about how it's derived.

Reply:
I mentioned this in the PR description, but this particular format was introduced back when the roles API controller was originally built. It never had an official name, so to differentiate it, I'm calling it the "explicit-scope format". This is in contrast to the current permission array format, where global is the implied scope on indexed array keys and per-resource permissions are associative array values.